### PR TITLE
Have region role wait for postgres election

### PR DESCRIPTION
**Pull request by cpg1111**
This requires https://github.com/maas/maas-ansible-playbook/pull/87 to land first

### CODE REVIEW

File: roles/maas_postgres/tasks/install_postgres.yaml [hunk 1]
```diff
@@ -82,11 +82,23 @@
   ansible.builtin.template:
     src: pgsql_check.j2
     dest: /opt/pgsql_check
-    mode: 0711
+    mode: 0755
```
Comment (alexsander-souza) [inline roles/maas_postgres/tasks/install_postgres.yaml:85]: this file has the postgres password in plain-text, does it needs to be world-readable?

**Review by alexsander-souza — APPROVED** [review]
+1, just a security concern about file permissions

File: roles/maas_postgres/tasks/install_postgres.yaml (hunk 1 above)
Comment (cpg1111) [reply roles/maas_postgres/tasks/install_postgres.yaml:85]: It does, xinetd runs as `nobody` and requires read access to the script before executing it.

Comment (alexsander-souza) [reply roles/maas_postgres/tasks/install_postgres.yaml:85]: :man_shrugging:

**Pull request merged**